packages/block-library/src/video/tracks-editor.js

```diff
@@ -99,8 +99,7 @@ function SingleTrackEditor( { track, onChange, onClose, onRemove } ) {
 				</span>
 				<Grid columns={ 2 } gap={ 4 }>
 					<TextControl
-						// TODO: Switch to `true` (40px size) if possible
-						__next40pxDefaultSize={ false }
+						__next40pxDefaultSize
```

Comment:
Can be seen when you add a text track to a Video block. Here is a test file to upload, if you're interested (save with a `.vtt` extension):

```
WEBVTT

00:00:00.500 --> 00:00:02.000
The Web is always changing

00:00:02.500 --> 00:00:04.300
and the way we access it is changing
```

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/ca78acc6-eb16-4a9e-928e-bcb3aae46168" alt="Video tracks editor, before" width="377">|<img src="https://github.com/user-attachments/assets/ef6c15c5-ea18-4994-aed2-0c97cf2559e2" alt="Video tracks editor, after" width="377">|